src/plugins/vis_type_timeseries/public/application/visualizations/views/timeseries/index.js

```diff
@@ -131,7 +132,7 @@ export const TimeSeries = ({
         baseTheme={theme}
         tooltip={{
           snap: true,
-          type: TooltipType.VerticalCursor,
+          type: tooltipMode === 'show_all' ? TooltipType.VerticalCursor : TooltipType.Follow,
```

Comment:
This check should most likely check for `show_focused` instead. Currently when you create a new chart, and never change that the tooltip mode, you'll have `undefined` as a value in there, and thus the FOLLOW mode will be used, while due to the `defaults` above we'll show the "Show all values" option in the select. But that defaults above is never written to the state, so we should make sure this check here treats undefined as `show_all` and not the other way around.

Steps to reproduce:

1) Create a new TSVB chart
2) Go to panel options
3) See that the tooltip mode is on "Show all value", but hovering over the chart will only show the single value focused.

Reply:
Great catch, thank you